### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by mrcuongnd**

### CODE REVIEW

**Comment by mrcuongnd** [comment]
create pull request test

**Comment by tucq88** [comment]
lol `test commit`? Do we need to merge this ?

**Review by vuhung — COMMENTED** [review]
Test commit thì OK rồi đấy @mrcuongnd

**Comment by vuhung** [comment]
@mrcuongnd Commit thật thôi. Test commit OK.

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+HTML5 Farming Demo created by Pablo Farias Navarro
+@ZenvaTweets
+www.pablofarias.com
+==================================================
+License of the code: GLP 3.0    http://opensource.org/licenses/gpl-3.0.html
```
Comment (hieulq) [inline readme.txt:5]: GLP?

**Comment by hieulq** [comment]
@mrcuongnd done testing, please submit a real PR.
Note that GPL v3 is not compatible with Apache 2.0 license, maybe we should change the license of this repo.

**Pull request closed without merging**